api/v1beta1/operatorpolicy_types.go

```diff
@@ -130,7 +133,7 @@ type OperatorPolicySpec struct {
 	// in 'inform' mode, and which installPlans are approved when in 'enforce' mode
 	Versions []policyv1.NonEmptyString `json:"versions,omitempty"`
 
-	//+kubebuilder:default={}
+	// +kubebuilder:default={}
```

Comment:
Generally, comments related to autogenerating code don't have a space after `//` but I'm not strict about this. Some others on the team may be :wink: